README.md

```diff
@@ -1,3 +1,16 @@
 # hamilflow
 
 Dataset of simple physical systems.
+
+## Publish process
+
+- Squash merge all features to `main`
+  - Use `feat(section): xxx`, `fix(section): yyy`, `chore(poetry): lock` etc as the squashed commit messages
+- Run `cz bump --dry` to determine the new version `X.Y.Z`. Or check out the [documentation](https://commitizen-tools.github.io/commitizen/bump/) for imposing a version
+- Run `git checkout release/X.Y.Z` (-b? not sure... I have always been using SourceTree )
+- Run `cz bump`. This will
+  - Write `docs/changelog.md` from the commit messages; update the version in `pyproject.toml`
+  - Commit `docs/changelog.md` and `pyproject.toml`
+  - Create a new tag
+- Push to `release/X.Y.Z`
+- Merge to `main`, no need to squash, maybe don't delete the branch under release
```

Comment:
> maybe don't delete the branch under release

This sounds redundant. 
I am quite confused by this. Tags will have exactly the release commit. What is the benefit of clustering the repo with branches. Maybe I missed something?

Reply:
- `cz` updates `changelog.md` and commits the change
- We then need to push the commit. If we don't create a branch `release/X.Y.Z`, we would have to push the commit directly to `main`.

Reply:
> * `cz` updates `changelog.md` and commits the change
> * We then need to push the commit. If we don't create a branch `release/X.Y.Z`, we would have to push the commit directly to `main`.

Can this stay in the same working branch? For example, this current branch `feature/18-use-commitizen`?